lib/syscall_shim/include/uk/syscall.h

```diff
@@ -69,22 +46,12 @@
 #include <stdarg.h>
 #include <uk/print.h>
 #include "legacy_syscall.h"
+#include "../../arch/regmap_linuxabi.h"
```

Comment:
if we need the linuxabi header here, better to move (officially export) the header to something like
`include/uk/syscall_shim/regmap_linuxabu.h`

Reply:
or actually, place it under the same path as the other syscall related sub-headers:
`<uk/bits/syscall_linuxabi.h>`